config.yaml

```diff
@@ -438,7 +438,7 @@ languages:
           weight: 90
         # LOGS - Subnav start
         - identifier: customnav_lognav_explore
-          name: "Explore"
+          name: "Search & Graph"
```

Comment:
search and graph seems odd. whats here?

Reply:
so there is a lot of stuff in search and graph. can it be split out? also the other menu items are processing and parsing. to keep it consistent, can it be searching and graphing? or change the others to process and parse

Reply:
Yup that's the end-goal of this, but since there is no real "Graph" i.e it's only the log graph timeseries that's available for now. We thought that it wouldn't be relevant to have one page for graphing almost empty.

But as soon as a new log graph is available, this content will be moved into `/logs/explore/log_graphs`